### PR TITLE
Feature/styling updates

### PR DESCRIPTION
**Pull request by aclavio**
Unified code styling with the combination of the eslint and prettier.js settings.

### CODE REVIEW

File: state-conductor-example/src/test/ml-modules/root/test/suites/StateConductorSuite/state-conductor-context-tests.sjs [hunk 1]
```diff
@@ -14,13 +14,18 @@ const doc2 = '/data/test-doc2.json';
...
 
 function isolate(func) {
-  return fn.head(xdmp.invokeFunction(() => {
-    declareUpdate();
-    return func();
-  }, {
-    isolation: 'different-transaction',
-    commit: 'auto'
-  }));
+  return fn.head(
```
Comment (freshie) [inline state-conductor-example/src/test/ml-modules/root/test/suites/StateConductorSuite/state-conductor-context-tests.sjs:17]: I dont like how it handles this I feel like it looked cleaner before. Is there away to change this?

File: state-conductor-modules/src/main/ml-modules/root/state-conductor/actions/common/dhf/dhf5MakeEnvelopeStep.sjs [hunk 2]
```diff
@@ -6,21 +6,28 @@ function performAction(uri, options = {}, context = {}) {
...
-    value: doc
+    value: doc,
   };
 
-  xdmp.log(Sequence.from([
-    'Execute DHF make envelope step:',
-    '  uri:     ' + uri,
-    '  options: ' + options,
-  ]), 'debug');
+  xdmp.log(
```
Comment (freshie) [inline state-conductor-modules/src/main/ml-modules/root/state-conductor/actions/common/dhf/dhf5MakeEnvelopeStep.sjs:12]: kind of the same thing for this one.

File: state-conductor-modules/src/main/ml-modules/services/state-conductor-resume.sjs [hunk 3]
```diff
@@ -19,28 +22,28 @@ function put(context, { uris = [], resumeBy = 'unspecified' }, input) {
...
-
-      resp.jobs[uri] = sc.resumeWaitingJob(uri, resumeBy);
-
-    });
-
-  }, {
-    database: xdmp.database(sc.STATE_CONDUCTOR_JOBS_DB)
-  })
+  sc.invokeOrApplyFunction(
+    () => {
```
Comment (freshie) [inline state-conductor-modules/src/main/ml-modules/services/state-conductor-resume.sjs:34]: it doesnt seem to handle the anonymous function well

File: state-conductor-modules/src/main/ml-modules/root/state-conductor/state-conductor.sjs [hunk 4]
```diff
@@ -151,7 +163,10 @@ function getFlowDocuments() {
  * @returns
  */
 function getFlowNames() {
-  return cts.uris('/', ['document'], cts.collectionQuery(FLOW_COLLECTION)).toArray().map(uri => getFlowNameFromUri(uri));
+  return cts
```
Comment (freshie) [inline state-conductor-modules/src/main/ml-modules/root/state-conductor/state-conductor.sjs:166]: ew.....

File: state-conductor-modules/src/main/ml-modules/root/state-conductor/state-conductor.sjs [hunk 5]
```diff
@@ -188,19 +208,20 @@ function getInitialState({ flowName, StartAt }) {
...
  */
 function getJobMetadatProperty(uri, flowName) {
-
   xdmp.securityAssert(
     'http://marklogic.com/state-conductor/privilege/execute',
     'execute'
   );
   if (fn.docAvailable(uri)) {
-    return xdmp.documentGetProperties(uri, fn.QName('', FLOW_JOBID_PROP_NAME))
+    return xdmp
```
Comment (freshie) [inline state-conductor-modules/src/main/ml-modules/root/state-conductor/state-conductor.sjs:216]: i'm not liking this splitting of a function name

File: state-conductor-modules/src/main/ml-modules/root/state-conductor/state-conductor.sjs [hunk 6]
```diff
@@ -213,7 +234,7 @@ function addJobMetadata(uri, flowName, jobId) {
   builder.startElement(FLOW_JOBID_PROP_NAME);
   builder.addAttribute('flow-name', flowName);
   builder.addAttribute('job-id', jobId);
-  builder.addAttribute('date', (new Date()).toISOString());
+  builder.addAttribute('date', new Date().toISOString());
```
Comment (freshie) [inline state-conductor-modules/src/main/ml-modules/root/state-conductor/state-conductor.sjs:237]: does this work the same?

**Comment by freshie** [comment]
Nothing is a show stopper for me on this. There are a few ways it formats things that is odd and I guess we will just have to get use to.

File: state-conductor-modules/src/main/ml-modules/root/state-conductor/state-conductor.sjs (hunk 6 above)
Comment (aclavio) [reply state-conductor-modules/src/main/ml-modules/root/state-conductor/state-conductor.sjs:237]: It does, though it looks odd

**Comment by aclavio** [comment]
Prettier is "opinionated code formatter" and doesn't provide a ton of configuration options.  The trade-off is for a uniform look in feel.

If there are specific lines/blocks of code we want it to ignore you can add a 
```
// prettier-ignore
```
comment and it won't be formatted

**Pull request merged**